pyproject.toml

```diff
@@ -196,7 +196,7 @@ oidc = ["authlib"]
 systemd = ["systemd-python"]
 url_preview = ["lxml"]
 sentry = ["sentry-sdk"]
-opentracing = ["jaeger-client", "opentracing"]
+opentelemetry = ["opentelemetry-api", "opentelemetry-sdk"]
```

Comment:
I don't know what I am doing with the dependencies here. Please double check this stuff.